mule-migration-tool-library/src/main/java/com/mulesoft/tools/migration/library/mule/steps/salesforce/CachedBasicConfiguration.java

```diff
@@ -42,7 +42,7 @@ public CachedBasicConfiguration() {
   @Override
   public void execute(Element mule3CachedBasicConfig, MigrationReport report) throws RuntimeException {
     addNameSpace(SalesforceUtils.MULE4_SALESFORCE_NAMESPACE,
-                                       SalesforceUtils.MULE4_SALESFORCE_NAMESPACE_URI, mule3CachedBasicConfig.getDocument());
+                 SalesforceUtils.MULE4_SALESFORCE_NAMESPACE_URI, mule3CachedBasicConfig.getDocument());
```

Comment:
Here goes the connector schema url instead of the uri. You should use this [constant](https://github.com/mulesoft/mule-migration-assistant/blob/d1f516afc3e7a4f01340742b46245420a7905ec0/mule-migration-tool-library/src/main/java/com/mulesoft/tools/migration/library/tools/SalesforceUtils.java#L35) instead.

Reply:
done